test/test_helpers/email_alert_api_test.rb

```diff
@@ -22,20 +22,6 @@
     end
   end
 
-  describe "#assert_email_alert_api_message_created" do
-    before { stub_any_email_alert_api_call }
-
-    it "matches a post request with any empty attributes by default" do
-      email_alert_api.create_message("foo" => "bar")
-      assert_email_alert_api_message_created
```

Comment:
Can we also remove this method declaration?
https://github.com/alphagov/gds-api-adapters/blob/54fed7b96da31a22b565965e03ac063a84a81cd3/lib/gds_api/test_helpers/email_alert_api.rb#L183